app/src/main/java/com/example/movieapp/ui/viewmodel/MainViewModel.kt

```diff
@@ -51,11 +55,17 @@ class MainViewModel : ViewModel() {
 
     val moviesViewHolderListener = object : MovieViewHolder.MoviesViewHolderListener {
```

Comment:
The viewModel should not do this. You have to make your fragment implement the ViewHolder interface the make the actions from the callback in your fragment

Reply:
dasn le fragment implémenter le view holder. adapter passe this